makerdao/governance-portal-v2
# fetch the delegates an address has delegated to

### Link to Shortcut ticket:

https://app.shortcut.com/dux-makerdao/story/765/add-delegated-mkr-to-list-to-account-page
### What does this PR do?

Adds MKR delegated to in the address detail page

### Steps for testing:

Go to a delegate, find people who delegated to him, click on the address and go to the detail, see MKR delegated to that address , time and transaction

### Screenshots (if relevant):

### Any additional helpful information?:

### Add a GIF:
